src/app/shared/components/entity-checkbox-dropdown/entity-checkbox-dropdown.component.html

```diff
@@ -18,6 +18,6 @@
         <span *ngIf="entityControl.value?.length === 1"> {{ entityControl.value[0].title }} </span>
       </div>
     </mat-select-trigger>
-    <mat-option class="dropdown-option" *ngFor="let entity of entities" [value]="entity">{{ entity.title }} </mat-option>
+    <mat-option class="dropdown-option" *ngFor="let entity of entities" [value]="entity" (click)="userInteracted = true">{{ entity.title }} </mat-option>
```

Comment:
do not assign variables directly in template, it is better to create a function for this, even if it is one line